composer.json

```diff
@@ -35,5 +35,6 @@
         "branch-alias": {
             "dev-master": "2.0-dev"
         }
-    }
+    },
+    "minimum-stability": "dev"
```

Comment:
Wouldn't it be better to change the version constraints to `~2.3|~3.0@dev` etc. instead?